packages/core/src/middleware/koa-log.ts

```diff
@@ -46,17 +46,31 @@ export default function koaLog<StateT, ContextT, ResponseBodyT>(): MiddlewareTyp
       logPayload = deepmerge(logPayload, payload);
     };
 
+    // eslint-disable-next-line @silverhand/fp/no-let
+    let error;
```

Comment:
could log(type, { result, error }) separately and call `saveLog` at finally.  Looks like let error can be avoided.

Reply:
Refactored. PTAL